cromshell

```diff
@@ -777,7 +844,7 @@ function status()
       # We could not find 'Fail' in our metadata, so our original status is correct. 
       turtle
     else
-      turtleDead
+			doomedLogo
```

Comment:
more tabs in the rest of the file

Reply:
Oh boy.  I forgot to de-tab the file.  Good catch.